.github/workflows/lint.yml

```diff
@@ -20,4 +20,5 @@ jobs:
           python -m pip install --upgrade pre-commit
           python -m pip install -e '.[d]'
       - name: Lint
-        run: pre-commit run --all-files --show-diff-on-failure
+        run: |
+          pre-commit run --all-files --show-diff-on-failure
```

Comment:
Why add the linebreak here? This does not have anything to do with the actual topic of the PR.